src/Samples/PortableTask/PortableTask.csproj

```diff
@@ -12,7 +12,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="15.5.180" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" GeneratePathProperty="true" />
   </ItemGroup>
   <Target Name="UpdateXlf" />
+  <Target Name="CopyMSBuildUtilitiesToTemp" AfterTargets="Build">
```

Comment:
Put a comment here and on the reference that this is only needed for MSBuild-repo tests. Right now it's in Samples, and this is not a sample-related thing.